### ISSUE TITLE
client.send_read_acknowldge failing for channels

### ISSUE THREAD

**Issue by NidzaKornjaca** [LonamiWebs/Telethon#525]
telegram_client.py
Offending code:
```
        entity = self.get_input_entity(entity)
        if entity == InputPeerChannel:
            return self(channels.ReadHistoryRequest(entity, max_id=max_id))
        else:
            return self(messages.ReadHistoryRequest(entity, max_id=max_id))
```
Fix that works for me:
```
        entity = self.get_input_entity(entity)
        if isinstance(entity, InputPeerChannel):
            return self(channels.ReadHistoryRequest(entity, max_id=max_id))
        else:
            return self(messages.ReadHistoryRequest(entity, max_id=max_id))
```

**Comment by Lonami** [comment]
Would you like to make a pull request?

**Comment by NidzaKornjaca** [comment]
Sure!

**Comment by Lonami** [comment]
The patch is tiny, you can probably do it from GitHub directly.

**Comment by Lonami** [comment]
Closed on #526.
